app/controllers/users/sessions_controller.rb

```diff
@@ -204,6 +205,32 @@ def override_csp_for_google_analytics
     def sign_in_params
       params[resource_name]&.permit(:email) if request.post?
     end
+
+    def check_password_compromised
+      return if current_user.check_password_compromised_at.present? ||
```

Comment:
These user model method calls need to be updated for the rename in 6d5ed636c691f27dbd8a99d71c1b6669635fbeb0.